.gitignore

```diff
@@ -11,6 +11,9 @@
 /log/*
 /tmp/*
 
+# Ignore simplecov files
+/coverage/*
```

Comment:
Would be good at some point to plumb this into the duty calc

Reply:
Do you mean CircleCi wise? Think Miguel is on it already.